packaging/rpm/ondemand.spec

```diff
@@ -315,7 +316,6 @@ touch %{_localstatedir}/www/ood/apps/sys/myjobs/tmp/restart.txt
 
 %dir %{_sysconfdir}/ood
 %dir %{_sysconfdir}/ood/config
-%dir %{_sysconfdir}/ood/config/ondemand.d
```

Comment:
You may need to keep this change I think but worth to see if the parent includes it too.

Reply:
Yea there are a few directories under `/etc/ood/config`. A quick glance at the `%dir` documentation read to me as `make sure this directory is removed when the package is removed` suggesting it'll also wipe the children.

Which is to say, I'm not sure if we need it - though I guess an rspec test for the same would tell us (not sure if we have one that tests directories get removed when you remove the package).